can-control.js

```diff
@@ -285,14 +284,13 @@ var Control = Construct.extend(
 
 			if(!element) {
 				dev.warn('can/control/control.js: Creating an instance of a named control without passing an element');
-
-				element = vDom().createElement('div');
+				return;
```

Comment:
I'm actually rethinking whether this is a good idea. If we `return` here, the rest of the lifecycle (`init`, etc) still happens and you get an instance of your Control back, but without most (but not all) functionality. I think this could be very confusing. Maybe it does make sense to just throw here. @obaidott what do you think?